lib/tests/2wp.js

```diff
@@ -679,6 +682,55 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a pegout with value exactly minimum', async () => {
+
+      // Arrange
+
+      // Create a pegin for the serder to ensure there is enough funds to pegout and because this is the natural process
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const peginValueInSatoshis = btcToSatoshis(0.5);
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, satoshisToBtc(peginValueInSatoshis));
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+
+      const initial2wpBalances = await get2wpBalances(rskTxHelper, btcTxHelper);
+      const initialSenderAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      const pegoutValueInRbtc = 0.005;
+
+      // Act
+
+      const pegoutTransaction = await sendTxToBridge(rskTxHelper, pegoutValueInRbtc, senderRecipientInfo.rskRecipientRskAddressInfo.address);
+
+      // Assert
+
+      let bridgeStateAfterPegoutCreation;
+
+      // Callback to get the bridge state after the pegout is created
+      const pegoutCreatedCallback = async () => {
+        bridgeStateAfterPegoutCreation = await getBridgeState(rskTxHelper.getClient());
+      };
+
+      const callbacks = {
+        pegoutCreatedCallback
+      };
+
+      await triggerRelease(rskTxHelpers, btcTxHelper, callbacks);
```

Comment:
For another story, but maybe we could get rid of the callbacks passed to `triggerRelease`? I think it adds confusion and are probably not needed now that we got rid of regression testing

Reply:
In this case it is necessary if we want to get the state of the Bridge right when the pegout is waiting for confirmations.